aten/src/ATen/detail/PrivateUse1HooksInterface.h

```diff
@@ -20,6 +22,31 @@ struct TORCH_API PrivateUse1HooksInterface {
   }
 
   virtual void initPrivateUse1() const {}
+  virtual void resizePrivateUse1Bytes(at::Storage &storage, size_t newsize) const {
```

Comment:
@albanD for thoughts

If we add this virtual method, can you make sure it's used over in the tests at `test/cpp_extensions/open_registration_extension.cpp`?

Reply:
We should test what we can test via that extension yes. If it requires more, it's ok to leave it untested for now and users of the backend can test it on their end.